client/cli/src/main.rs

```diff
@@ -738,5 +761,43 @@ async fn handle_command(
                 Some(e.into()),
             )),
         },
+        Command::DecodeTransaction { hex_string } => {
+            let decoders = ModuleDecoderRegistry::from_iter([
```

Comment:
The correct way would be #1931 , but just using `module_decode_stubs` would be a good start.

Reply:
thanks, fixed it